### PR TITLE
updated patterns and partitioning

### PR DESCRIPTION
**Pull request by mslacken**
- changed required pattern from minimal_base to base bsc#1090282 
- added subvolume description to installation.xml to make btrfs
  available on guided partitioner (bsc#1090380)
- changed workload_server to hpc_workload_server (bsc#108951)

### CODE REVIEW

File: control/installation.xml [hunk 1]
```diff
@@ -108,8 +162,9 @@
                   </volumes>
                 </partitioning>
                 <software>
-		  <!-- we do need most likely more repos with "base ofed"-->
-                  <default_patterns>minimal_base hpc_compute_node hpc_libraries ofed file_server hpc_development_node yast2_basis devel_basis workload_server</default_patterns>
+		  <!-- using base as minimal_base would nearly provide as much packages as
```
Comment (imobachgs) [inline control/installation.xml]: The comment looks confusing to me (maybe it is just me). What I read is:  *"base" would nearly provide as much packages as "base"*.

File: control/installation.xml [hunk 2]
```diff
@@ -65,15 +65,69 @@
                     <volume>
                       <mount_point>/</mount_point>
                       <fs_type>xfs</fs_type>
-                      <fs_types>xfs,ext3,ext4</fs_types>
+                      <fs_types>xfs,ext3,ext4,btrfs</fs_types>
                       <desired_size config:type="disksize">16 GiB</desired_size>
                       <min_size config:type="disksize">4 GiB</min_size>
                       <max_size config:type="disksize">32 GiB</max_size>
                       <weight config:type="integer">20</weight>
+			<snapshots config:type="boolean">true</snapshots>
```
Comment (imobachgs) [inline control/installation.xml]: Please, check indentation

**Review by imobachgs — CHANGES_REQUESTED** [review]

File: control/installation.xml (hunk 1 above)
Comment (mslacken) [reply control/installation.xml]: Should be: using base instead of mininmal_base

File: control/installation.xml (hunk 2 above)
Comment (mslacken) [reply control/installation.xml]: are there rules for that? vim command would also be fine.

**Review by imobachgs — APPROVED** [review]

**Pull request merged**